compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/AggregatedOpInterfaceImpl.cpp

```diff
@@ -20,6 +21,13 @@
 
 namespace mlir::iree_compiler::IREE::LinalgExt {
 
+/// Command line options used purely for development purposes. Not to be relied
```

Comment:
NIT: Can we add clarifying comment that this is for numerics/quantization, since technically `maximum expected value from attention softmax` is always 1 no? haha

Reply:
Updated comment.